src/ui/utils.js

```diff
@@ -45,11 +46,40 @@ export const amountStyle = (amount) => {
 
 export const insertIf = (condition, ...elements) => (condition ? elements : [])
 
-export const fixedFloat = val => val && val.toFixed(6)
+export const fixedFloat = val => (typeof val === 'number' ? val && val.toFixed(8) : val)
```

Comment:
val is already a number, so it is already guarantee to have `toFixed` func ?

Reply:
just a check to display 0 without eight nulls